packages/host/app/components/ai-assistant/panel-popover.gts

```diff
@@ -0,0 +1,45 @@
+import Component from '@glimmer/component';
+
+interface Signature {
+  Element: HTMLElement;
+  Blocks: {
+    header: [];
+    body: [];
+  };
+}
+
+export default class AiAssistantPanelPopover extends Component<Signature> {
```

Comment:
This can be reused when we add other popovers such as "Rename session":

<img width="456" alt="image" src="https://github.com/cardstack/boxel/assets/273660/882e1cf9-5878-4cdc-95d3-839a60b49d94">